### DISCUSSION TITLE
Support XDelta1 patches

### DISCUSSION THREAD

**Discussion by btimofeev** [btimofeev/UniPatcher#1] [Ideas]

**Comment by tjoeylc** [comment]
Unipatcher would be much more useful with this.

**Comment by btimofeev** [comment]
Yes, I will try to add this.  I already tried before, but failed.  But it seems I found what I had a problem with, so maybe soon the application will support this format.

**Comment by btimofeev** [comment]
Yesterday I compiled the xdelta1 code (I borrowed this from @xperia64 https://github.com/xperia64/android-rom-patcher). It works if compiled to a 32 bit library, but crashes in 64 bit. 

And since Google Play now requires the mandatory presence of a 64-bit library in the APK, I cannot release the application in this state.

I may have to build 32-bit versions and only release it to Github and F-Droid if I can't solve this problem in the future.
